src/test/java/com/saucelabs/ci/sauceconnect/SauceConnectFourManagerTest.java

```diff
@@ -119,35 +133,39 @@ public void openConnectionTest_existing_tunnel() throws Exception {
     public TemporaryFolder folder= new TemporaryFolder();
 
     @Test
-    public void testExtractZipFile() throws Exception {
+    public void testExtractZipFileWithoutCleanUpOnExit() throws Exception {
+        testExtractZipFile(false);
+    }
+
+    @Test
+    public void testExtractZipFileWithCleanUpOnExit() throws Exception {
+        testExtractZipFile(false);
```

Comment:
Shouldn't it be 
```suggestion
        testExtractZipFile(true);
```

The fact of removing the particular files should be tested as well.